ibm/service/vpc/resource_ibm_is_vpc_routing_table_route.go

```diff
@@ -66,9 +66,9 @@ func ResourceIBMISVPCRoutingTableRoute() *schema.Resource {
 				Description: "The zone to apply the route to. Traffic from subnets in this zone will be subject to this route.",
 			},
 			rNextHop: {
-				Type:        schema.TypeString,
-				Required:    true,
-				ForceNew:    true,
+				Type:     schema.TypeString,
+				Required: true, // recheck as the generated says it as, not required.
```

Comment:
I see it is not required field in the spec, but if it is required already we can leave it as it is.